### PR TITLE
Update (broken) links to INBO package

### PR DESCRIPTION
**Pull request by peterdesmet**
- protocolhelper: link to website, not repo
- datapackage: added
- pyinaturalist: link to website, not repo
- bioRad: added
- git2rdata: update broken link
- camtrap: added

### CODE REVIEW

**Comment by peterdesmet** [comment]
@hansvancalster maybe I should also describe them in the text below?

**Comment by florisvdh** [comment]
> maybe I should also describe them in the text below?

Preferably, yes!

Thanks for updating the URLs; can you update them also in the descriptions? Maybe in the future we can have a better way to automate this.

**Review by hansvancalster — APPROVED** [review]
Thanks for updating. And indeed best change the links in the text as well (which for instance still refers to `uvabits`) before merging. Approving already so you can proceed.

**Comment by florisvdh** [comment]
@peterdesmet just a reminder to take up the remaining job in this PR. This to avoid later conflicts with PRs that act on the same file, and are waiting to be merged e.g. #293.
Thanks!

**Comment by peterdesmet** [comment]
- Made references for package links at the top, which then can be used throughout the document
- Sorted packages alphabetically within section
- Added descriptions for bioRad, frictionless, gulltracking and camtrapdp
- Added frictionless package (twice though, as it can be used to retrieve data and store data

@hansvancalster @florisvdh up to you if you want to re-review this?

File: content/articles/inbo_software/index.md
Comment (hansvancalster) [inline content/articles/inbo_software/index.md:66]:
```suggestion
- **R package [protocolhelper][protocolhelper]**: provides templates for protocols and helper functions to start developing a new protocol or move an existing protocol to the [INBO protocols repository](https://github.com/inbo/protocolsource)
```

**Review by hansvancalster — COMMENTED** [review]
Thanks @peterdesmet . 👍 for defining the links to webpages beforehand. Makes it easier to read.

**Comment by peterdesmet** [comment]
One last note: the listed `grts` package is apparently superseeded by `grtsdb`. I have not updated that (it is clearly indicated if you go to https://github.com/ThierryO/grts), but maybe it should. @ThierryO I'll leave that up to you.

**Comment by florisvdh** [comment]
Nice!

**Pull request merged**
